src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostRawUserInterface.cs

```diff
@@ -17,6 +17,8 @@
 using WORD = System.UInt16;
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Test-Connection.Ping.Force IPv4 with implicit PingOptions
```powershell
Expected exactly '10.79.2.17', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 84
84:             $result[0].Address | Should -BeExactly $realAddress
```
Test-Connection.TraceRoute.TraceRoute works
```powershell
Expected exactly '10.79.2.17', but got 127.0.0.1.
at <ScriptBlock>, /Users/runner/runners/2.165.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 258
258:             $result[0].TargetAddress | Should -BeExactly $realAddress
```